lms/djangoapps/verify_student/views.py

```diff
@@ -895,8 +895,8 @@ def _compose_message_reverification_email(
         reverification_block = modulestore().get_item(usage_key)
         # Allowed attempts is 1 if not set on verification block
         allowed_attempts = 1 if reverification_block.attempts == 0 else reverification_block.attempts
-        user_attempts = VerificationStatus.get_user_attempts(user_id, course_key, relates_assessment, location_id)
-        left_attempts = allowed_attempts - user_attempts
+        used_attempts = VerificationStatus.get_user_attempts(user_id, course_key, relates_assessment, location_id)
+        left_attempts = allowed_attempts - used_attempts
```

Comment:
Looking through the code, I'm confused by this as well.  What I'm seeing in the reverification block code doesn't match what I remember we discussed with Griff.

The value that course teams enter in Studio means the number of *additional* re-verification attempts after the first one.  So if it's `0`, that means that means the user gets one re-verification attempt by default.  If it's `1`, then the user gets two attempts (the first plus one additional).  And so on (2 --> 3, 3 --> 4).

Does that make sense?